src/main/resources/mockdata/05bdf3c8-01f0-4ddb-bd6c-6efd465f9e33.json

```diff
@@ -1,154 +1,172 @@
+        "license": "09cld8be-cabe-11e8-a8d5-f28201f1b9fd1"
       },
+      "fund_distribution": [
+        "UNIV-SUBN-FY19"
+      ],
       "location": {
-        "id": "07c60ea6-cabe-11e8-a8d5-f2801f1b9fd1",
+        "id": "07v5d9ea-cabe-11e8-a8d5-f2801f1b9fd1",
+        "location_id": "ncc5d9ea-cabe-11e8-a8d5-f2901f1b9fd2",
+        "po_line_id": "c7c5d04e-c123-1132-87d5-f2801f1b9fd1",
         "quantity": 2,
-        "location_id": "stks,www"
-      },
-      "order_format": {
-        "id": "07c60fd2-cabe-11e8-a8d5-f2801f1b9fd1",
-        "code": null,
-        "description": "P/E Mix"
-      },
-      "order_type": {
-        "id": "07c610fe-cabe-11e8-a8d5-f2801f1b9fd1",
-        "code": null,
-        "description": "On-time"
+        "quantity_electronic": 1,
+        "quantity_physical": 1
       },
+      "order_format": "P/E Mix",
+      "owner": "Cushing",
+      "payment_status": "Pending",
       "physical": {
-        "id": "07c61284-cabe-11e8-a8d5-f2801f1b9fd1",
-        "expected_receipt_date": null,
-        "receipt_due": "2018-08-19T10:00:00.000+0000",
-        "expected_receipt_interval": 0,
+        "id": null,
+        "material_supplier": "Ro*Co films",
+        "receipt_due": "2018-08-19T00:00:00.000Z",
         "volumes": 1,
-        "material_id": "",
-        "material_supplier": "Ro*Co films"
-      },
-      "purchase_order_id": "",
-      "receipt_status": {
-        "id": "07c613c4-cabe-11e8-a8d5-f2801f1b9fd1",
-        "code": null,
-        "description": "Received"
+        "po_line_id": null
       },
+      "po_line_description": "",
+      "po_line_number": "1",
+      "po_line_workflow_status": "Open",
+      "publication_date": "2016",
+      "publisher": "ro*co films",
+      "purchase_order_id": "c79b0bcc-DcAD-1E4E-Abb7-DbFcaD5BB3bb",
+      "receipt_date": null,
+      "receipt_status": "Awaiting Receipt",
       "renewal": {
-        "id": "07c6170c-cabe-11e8-a8d5-f2801f1b9fd1",
-        "cycle": "manual",
-        "review_period": 90,
-        "renewal_date": "2021-07-19T10:00:00.000+0000",
+        "id": "9e414baa-8cf2-4916-a90f-d1da4009e09e",
+        "cycle": "Manual",
         "interval": 0,
-        "manual_renewal": true
+        "manual_renewal": true,
+        "review_period": 90,
+        "renewal_date": "2018-07-19T00:00:00.000Z",
+        "po_line_id": "07c5d04e-c123-1132-87d5-f2801f1b9fc1"
       },
+      "reporting_codes": [
+        {
+          "id": "59262cd7-85f5-450b-8283-712595ebb38b",
+          "code": "CODE1",
+          "description": ""
+        },
+        {
+          "id": "fa326c04-81n1-4b72-8aaf-01281bac718f",
+          "code": "CODE2",
+          "description": ""
+        }
+      ],
+      "requester": "jcobb",
+      "rush": false,
+      "selector": "mreynolds",
       "source": {
-        "id": "07c6186a-cabe-11e8-a8d5-f2801f1b9fd1",
-        "code": null,
+        "id": "0b4b2f41-v5c6-4280-858e-33fba452a334",
+        "code": "",
         "description": "manual"
```

Comment:
Manually entered.

Reply:
What do you mean? should I fill `"code": "manual"` and `"description": "Manually entered"` or leave them `" "` or `null`. What about `"vendor"` => should it be e.g. `"Amazon.com"` or UUID?

Reply:
`"description": "Manually entered"`  Sorry I wasn't clear

Reply:
>  What about `"vendor"` => should it be e.g. "Amazon.com" or UUID?

Reply:
UUID